### PR TITLE
updates the environment variables table to login to dockerhub

### PR DESCRIPTION
**Pull request by vjayaramrh**
This PR is related to the changes made in https://github.com/nephio-project/test-infra/pull/182

### CODE REVIEW

File: install-guide/README.md [hunk 1]
```diff
@@ -112,15 +112,17 @@ sudo NEPHIO_DEBUG=false   \
...
-| NEPHIO_REPO            | URL              | https://github.com/nephio-project/test-infra.git |URL of the repository to be used for installation |
-| NEPHIO_BRANCH          | branch or tag    | main          | Tag or branch name to use in NEPHIO_REPO |
+| NEPHIO_DEBUG           | false or true    | false         | Controls debug output from the install                                       |
+| NEPHIO_HOME            | path             | /home/$NEPHIO_USER | The directory to check out the install scripts into                          |
+| NEPHIO_DEPLOYMENT_TYPE | r1 or one-summit | r1            | Controls the type of installation to be carried out                          |
+| RUN_E2E                | false or true    | false         | Specifies whether end-to-end tests should be executed or not                 |
+| DOCKERHUB_USERNAME     | alpha-num string |               | Specifies the dockerhub username                                             |
+ | DOCKERHUB_TOKEN       | alpha-num string |               | Specifies the password or token                                              |
```
Comment (electrocucaracha) [inline install-guide/README.md:123]:
```suggestion
| DOCKERHUB_TOKEN        | alpha-num string |               | Specifies the password or token                                              |
```

**Review by electrocucaracha — COMMENTED** [review]
Let's sync this change until the another PR gets merged

**Comment by electrocucaracha** [comment]
@vjayaramrh could you rebase this patch for merging it?

**Comment by vjayaramrh** [comment]
> @vjayaramrh could you rebase this patch for merging it?

Thanks @electrocucaracha , Its rebased now.

**Pull request merged**